src/lib/auth.ts

```diff
@@ -57,58 +63,75 @@ export const ROLES: {
 };
 
 export const DEVICE_API_KEY_PERMISSIONS = [
-	`resin.device_type.get?describes_device/any(d:d/${matchesActor})`,
-	`resin.device.get?${matchesActor}`,
+	`resin.device_type.read?describes_device/canAccess()`,
```

Comment:
This technically expands the permissions to also be able to read device types of dependent devices, but I think that's actually correct